pom.xml

```diff
@@ -663,7 +663,7 @@
                             <execution>
                                 <id>add-profile-src-31+</id>
                                 <goals><goal>add-source</goal></goals>
-                                <phase>generate-sources</phase>
+                                <phase>none</phase>  <!--Disable it for all submodules, only sql-plugin needs to enable it -->
```

Comment:
How about removing this whole execution if it was not needed ?

Reply:
You are right. But not sure if submodules will use this in future.  
Let's keep it in the parent pom.